libGeoIP/GeoIP.h

```diff
@@ -33,7 +33,11 @@ extern "C" {
 #else /* !defined(_WIN32) */
 #include <winsock2.h>
 #include <ws2tcpip.h>
-#define snprintf _snprintf
+
+#if defined(_MSC_VER) && _MSC_VER < 1900
+#  define snprintf _snprintf
```

Comment:
It seems like we should move this into GeoIP.c rather than defining it in the header file.